### PR TITLE
initial version of MCS setup guide and samples 

### PR DESCRIPTION
**Pull request by luolanzone**
Signed-off-by: Lan Luo <luola@vmware.com>

### CODE REVIEW

**Comment by luolanzone** [comment]
Hi @aravindakidambi  here is the initial version of MCS docs which I added in https://github.com/antrea-io/antrea/pull/2844, I removed the cert-manager related lines, you may continue to work on this. thanks!

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md]: This gives the impression that member clusters must use the same namespace as the leader, so maybe we should not use a namespace here?

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md]: this should be antrea-mcs-ns

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md]: we dont need these for the cluster-claim in the leader

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md:169]: We need a secret account name that is associated with these members

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md:175]: Need to add help about creating sa and associated role-binding, i can help with that after this is merged

**Review by aravindakidambi — APPROVED** [review]

File: multicluster/docs/getting-started.md
Comment (luolanzone) [reply multicluster/docs/getting-started.md]: ClusterClaim is namespaced, you mean leave it empty to create it in `default` namespace?

File: multicluster/docs/getting-started.md
Comment (luolanzone) [reply multicluster/docs/getting-started.md]: you mean `secret` and `server`?

File: multicluster/docs/getting-started.md
Comment (luolanzone) [reply multicluster/docs/getting-started.md:169]: it can work fine without secret account in my local env, so I am wondering how this secret account will be used in leader cluster?

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [reply multicluster/docs/getting-started.md]: yes, using the same ns as the one in leader cluster may seem like it needs to be in the same ns

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [reply multicluster/docs/getting-started.md]: Yeah on leader cluster, we dont need secret and server IP right?

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [reply multicluster/docs/getting-started.md:169]: thats right, i think we dont have webhook yet, eventually we want webhook to validate that member id and secret account matches for incoming cruds

File: multicluster/docs/getting-started.md
Comment (luolanzone) [reply multicluster/docs/getting-started.md:175]: hi @aravindakidambi  I have addressed your comments, please help to double check, if it's ok, I think you can merge and update sa and role-binding parts. thanks.

File: multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml
Comment (aravindakidambi) [inline multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml:9]: Do we need this? Because we need one for each member, I dint want to create this in the manifest by default?

File: multicluster/config/overlays/leader-ns/member_cluster_rolebinding.yaml
Comment (aravindakidambi) [inline multicluster/config/overlays/leader-ns/member_cluster_rolebinding.yaml:12]: newline

File: multicluster/config/overlays/leader-ns/member_cluster_serviceaccount.yaml
Comment (aravindakidambi) [inline multicluster/config/overlays/leader-ns/member_cluster_serviceaccount.yaml:5]: newline

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md:58]: I think we also need to edit it to remove annotations and change type to "mcs-custom" ?

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md:34]: I prefer we give them the example on how to manually create the SA and the RoleBinding rather than put it in the manifest. But I can change it after you merge this if you think its reasonable

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md:126]: default?

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md:92]: default?

File: multicluster/docs/getting-started.md
Comment (aravindakidambi) [inline multicluster/docs/getting-started.md:170]: But this SA name doesnt match the one in manifest member-access-sa. Thats why it maybe best to give the steps to create SA and role binding and not put it in the manifest?

**Pull request merged**
